examples/counterfleetautoscaler.yaml

```
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# [Stage:Alpha]
+# [FeatureFlag:CountsAndLists]
+# Example of a FleetAutoscaler - this is used to scale a Fleet automatically depending on the
+# available capacity of a Counter across the fleet.
+#
+
+#
+# For a full reference and details: https://agones.dev/site/docs/reference/fleetautoscaler/
+#
+
+apiVersion: "autoscaling.agones.dev/v1"
+kind: FleetAutoscaler
+metadata:
+ name: fleet-autoscaler-counter
+spec:
+ fleetName: fleet-example
+ policy:
+   type: Counter # Counter based autoscaling
+   counter:
+     # The name of the Counter.
+     # Required.
+     key: players
+     # Size of a buffer of counted items that are available in the Fleet (available capacity).
+     # bufferCount can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%).
+     # Required.
+     bufferCount: 5
```

I'm wondering 🤔  Should this be:

```suggestion
     bufferSize: 5
```

To match the `Buffer` auto scaling strategy, or should it be:

```suggestion
     bufferCapacity: 5
```

To match `min/maxCapacity` ?

Now that we changed `min/max` this feels out of place, and that way we could also be the same across `Count` and `List` based scaling (and we could consolidate Go types as well)

WDYT?

---

Makes sense, I made the change to bufferSize.